lib/model.js

```diff
@@ -1071,11 +1072,20 @@ class Model extends Entity {
                  */
                 if (typeof validationRule === 'object') {
```

Comment:
If I understand well we now have both 'object' and 'function' to validate (I just realized that the Readme has an error for the ip example). Like this

```js
ip  : {
    validate: {
        rule: 'isIP',
        args: [4]
    },
}

...

accessList: {
    validate: validateAccessList
}
```

Shouldn't this be written like this ?

```js
if (typeof validationRule === 'object') {
    validationRule = schema.paths[k].validate.rule;
    validationArgs = validationArgs.concat(schema.paths[k].validate.args || []);
    validationFn = validator[validationRule];
} else if (typeof validationRule === 'function') {
    validationFn = validationRule;
    validationArgs.push(validator);
} else {
    validationFn = validator[validationRule];
}
```

I might be missing something, it is just by reading the code :)